fem/quadinterpolator.hpp

```diff
@@ -99,26 +105,36 @@ class QuadratureInterpolator
              Vector &q_val, Vector &q_der, Vector &q_det) const;
 
    /// Interpolate the values of the E-vector @a e_vec at quadrature points.
+   template <QVectorLayout>
+   void Values(const Vector &e_vec, Vector &q_val) const;
    void Values(const Vector &e_vec, Vector &q_val) const;
```

Comment:
It is good to have separate documentation for these two methods to clarify things -- e.g. I assume the first one ignores the `q_layout` setting?

Reply:
Moved to the `Compute kernels` section with separate documentation.